src/pages/components/RevokeActionsButton.js

```diff
@@ -1,37 +1,214 @@
+/* eslint-disable prettier/prettier */
 import React from 'react'
 import styled, { keyframes } from 'styled-components'
 import { Button } from '@digicatapult/ui-component-library'
 
-const reasonsDummyJSON = {
-  predefinedReasons: {
-    0: { selection: null },
-    1: { selection: null },
-    2: { selection: null },
-  },
+import { useState, useCallback, useMemo } from 'react'
+import { useRef } from 'react'
+import { Dialog } from '@digicatapult/ui-component-library'
+
+import { Section } from '@digicatapult/ui-component-library'
+import WarningSignSvg from '../../assets/images/warning-sign-icon.svg'
```

Comment:
should this be in the ui component library?

Reply:
Indeed, I was thinking about that as well .. 

the problem is : 

I don't know what type UI element this is .. is this a background icon , is this an image for the beginning of a paragraph.. is this a text with a custom image bullet-point ? ( not to mention the time constrain )

I feel like this is more like a question for others ..